packages/core-components/src/layout/Content/Content.tsx

```diff
@@ -53,6 +53,14 @@ type Props = {
   className?: string;
 };
 
+/**
+ * HTML article tag wrapped with text alignment properties and custom className
```

Comment:
Hm let's not describe the implementation but rather the purpose. Suggestion: `The main content part inside a {@link Page}.`